File: 1.2 Patterns/17-tri-with-alpha-3.cpp
#include <iostream>
using namespace std;
int main()
{
    int row = 5, i, j, k;
    for (i = 0; i < row; i++)
    {
        for (j = 0; j < row - i; j++)
        {
            cout << " ";
        }
        for (k = 0; k < 2 * i + 1; k++)
        {
            cout << char(k + 65);
        }
        cout << "\n";
    }
    return 0;
}

File: 1.2 Patterns/17-tri-with-alpha-3.py
row = 5
i = 0
j = 0
k = 0
for i in range(0, row):
    for j in range(0, row - i):
        print(" ", end="")
    for k in range(0, 2 * i + 1):
        print(chr(k+65), end="")
    print("")                           # \n is inserted by default
